libs/ledger-live-common/src/apps/hw.ts

```diff
@@ -24,8 +23,9 @@ export const execWithTransport =
  *   currency and parent application. And therefor we no longer need the version
  *   check that broke that dependency after a certain version for ETH and BTC.
  * - Remove all the legacy v1 code, and tests.
- * - Remove the env definition and all the forks based on it here.
+ * - Cleanup the feature flag that governs this.
  */
-// Nb Written this way to respect runtime changes to the env.
+let listAppsV2Enabled = false;
```

Comment:
is there another way not to have a local state living in the module? couldn't an env still be used but the featureFlag effect will just `setEnv` it? 🤔

Reply:
Considering the idea is to drop the fork in the logic between v1 and v2 pretty soon and not leave this permanently, I'd argue it's acceptable to have a local state for a couple of weeks.

Reply:
ok, could a jira ticket be created about this + with a comment of that ticket link? 👍